### PR TITLE
[TR] Fix CI master

### PR DESCRIPTION
**Pull request by jjanvier**
| Q                  | A
| ------------------ | ---
| Migration script?  | -
| Behats             | running
| Specs & Static     | Y
| Changelog updated  | -

### CODE REVIEW

**Comment by nidup** [comment]
GTM

File: src/Pim/Component/Catalog/Updater/Copier/MediaAttributeCopier.php [hunk 1]
```diff
@@ -114,12 +114,17 @@ protected function copySingleValue(
...
+            if (null !== $fromValue->getMedia()) {
+                $filesystem = $this->mountManager->getFilesystem(FileStorage::CATALOG_STORAGE_ALIAS);
 
-            $rawFile = $this->rawFileFetcher->fetch($filesystem, $fromValue->getMedia()->getKey());
-            $file = $this->rawFileStorer->store($rawFile, FileStorage::CATALOG_STORAGE_ALIAS, false);
+                $rawFile = $this->rawFileFetcher->fetch($filesystem, $fromValue->getMedia()->getKey());
+                $file = $this->rawFileStorer->store($rawFile, FileStorage::CATALOG_STORAGE_ALIAS, false);
+
+                $file->setOriginalFilename($fromValue->getMedia()->getOriginalFilename());
+            } else {
```
Comment (ClementGautier) [inline src/Pim/Component/Catalog/Updater/Copier/MediaAttributeCopier.php]: If you want to skip the else statement you can instantiate $file to null before the if statement.

File: src/Pim/Component/Catalog/Updater/Copier/MediaAttributeCopier.php [hunk 2]
```diff
@@ -114,12 +114,17 @@ protected function copySingleValue(
...
             }
 
-            $filesystem = $this->mountManager->getFilesystem(FileStorage::CATALOG_STORAGE_ALIAS);
+            if (null !== $fromValue->getMedia()) {
+                $filesystem = $this->mountManager->getFilesystem(FileStorage::CATALOG_STORAGE_ALIAS);
 
-            $rawFile = $this->rawFileFetcher->fetch($filesystem, $fromValue->getMedia()->getKey());
-            $file = $this->rawFileStorer->store($rawFile, FileStorage::CATALOG_STORAGE_ALIAS, false);
+                $rawFile = $this->rawFileFetcher->fetch($filesystem, $fromValue->getMedia()->getKey());
+                $file = $this->rawFileStorer->store($rawFile, FileStorage::CATALOG_STORAGE_ALIAS, false);
```
Comment (phaseinducer) [inline src/Pim/Component/Catalog/Updater/Copier/MediaAttributeCopier.php]: align please

**Comment by phaseinducer** [comment]
litlle comment & GTM :+1:

**Pull request merged**